decidim-comments/app/packs/src/decidim/comments/comments.component.js

```diff
@@ -330,8 +330,6 @@ export default class CommentsComponent {
     if (!orderSelect) {
       return;
     }
-    orderSelect.style.fontWeight = "bold";
-    orderSelect.style.borderColor = "black";
```

Comment:
Fantastic! Lets keep line 334 which is the border-color of the select dropdown, not affecting the font-weight! Aside from that, its great!